foxtrot-core/src/main/java/com/flipkart/foxtrot/core/common/Action.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright 2014 Flipkart Internet Pvt. Ltd.
- *
+ * <p/>
```

Comment:
Remove all such lines. No HTML formatting in source code. Atleast not in the license section.